docs/book/src/developer/tilt.md

```diff
@@ -186,8 +186,19 @@ tilt up
 
 This will open the command-line HUD as well as a web browser interface. You can monitor Tilt's status in either
 location. After a brief amount of time, you should have a running development environment, and you should now be able to
-create a cluster. Please see the [Usage section in the Quick
-Start](https://cluster-api.sigs.k8s.io/user/quick-start.html#usage) for more information on creating workload clusters.
+create a cluster. There are [example worker cluster
+configs](https://github.com/kubernetes-sigs/cluster-api/tree/master/test/infrastructure/docker/examples) available.
+These can be customized for your specific needs.
+
+<aside class="note">
+
+<h1>Use of clusterctl</h1>
+
+When the worker cluster has been created using tilt, `clusterctl` should not be used for management
+operations. `clusterctl` is not aware of local code changes and will attempt to use
+public release versions for any updates.
```

Comment:
```suggestion
operations; this is because tilt doesn't initialize providers on the management cluster like clusterctl init does, so some of the clusterctl commands like clusterctl config won't work.

This limitation is a acceptable trade-off while executing fast dev-test iterations on controllers logic; if instead you are interested in testing clusterctl workflows, you should refer to the [clusterctl developer instructions](https://cluster-api.sigs.k8s.io/clusterctl/developers.html).
```